tests/components/orderentry/PastOrder.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Why have you tested a component that you didn't update? I thought this is part od OEUI-109?

Reply:
This pull request implementation is for past orders so it directly relates to the file I tested. Also, Coveralls flagged a red on the file which made the test coverage drop.

Reply:
I think you should test PastOrders file since that is the file whose testing has declined.

Reply:
And it will affect the total test coverage. PastOrder file has been tested in OEUI-109 according to the ticket description.